py/private/pytest_shard/LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright 2019 Adam Gleave
```

Comment:
also add a README pointing to the precise location you vendored these files from, so we can later reason about the delta from upstream changes